chord_metadata_service/phenopackets/summaries.py

```diff
@@ -77,8 +81,20 @@ async def phenotypic_feature_summary(
 
 
 async def dt_phenopacket_summary(
-    phenopackets: QuerySet, discovery: DiscoveryConfig, phenopacket_permissions: DataPermissionsDict
+    scope: ValidatedDiscoveryScope, phenopacket_permissions: DataPermissionsDict, queryset: QuerySet | None = None
 ) -> dict:
+    discovery = scope.discovery
+
+    # Start with either all phenopackets or a subset specified by a parameter
+    phenopackets = queryset if queryset is not None else models.Phenopacket.objects.all()
+
+    # Apply scope to existing queryset to enforce it on the summarization
+    if dataset_id := scope.dataset_id:
+        phenopackets = phenopackets.filter(dataset_id=dataset_id)
+    elif project_id := scope.project_id:
+        # Project is set but dataset isn't
+        phenopackets = phenopackets.select_related("project").filter(dataset__project_id=project_id)
```

Comment:
Shouldn't this be `select_related("dataset__project")` ?

Reply:
should be `dataset` i think, but good point